.gitignore

```diff
@@ -0,0 +1 @@
+node_modules/
```

Comment:
- [ ] Please include the `test.md` to the `.gitignore` file so that its content can be ignored by git so it doesn't get pushed to GitHub. This was a requirement 👍 💯